### DISCUSSION TITLE
Provide CMake build and install steps (preferably for each OS platform)

### DISCUSSION THREAD

**Discussion by jimjamz** [86Box/86Box#4710] [General]
The repository does not have any described build and install steps with the README.

I'm trying to build and install the project with cmake on Debian 12:
`cmake -DCMAKE_BUILD_TYPE=Release -S . -B ~/opt/86Box/build/`:

I first ran into the following issue:
```
CMake Error at /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Freetype (missing: FREETYPE_LIBRARY FREETYPE_INCLUDE_DIRS)
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.25/Modules/FindFreetype.cmake:162 (find_package_handle_standard_args)
  src/CMakeLists.txt:111 (find_package)
```

After installing the `libfreetype-dev` and `libfreetype6` packages, I then ran into this non-intuitive cmake error:
```
CMake Error at /usr/share/cmake-3.25/Modules/FindPkgConfig.cmake:607 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/FindPkgConfig.cmake:829 (_pkg_check_modules_internal)
  src/cdrom/CMakeLists.txt:18 (pkg_check_modules)
```

If you could perhaps provide some cmake build and install steps in the main repository README, it may help to alleviate such issues.

**Comment by seal331** [comment]
not a dev but this can help you: [https://86box.readthedocs.io/en/latest/dev/buildguide.html](https://86box.readthedocs.io/en/latest/dev/buildguide.html)

**Comment by jimjamz** [comment]
Excellent, thank you @seal331 .   I wasn't aware of this documentation.  It helped me to build the binaries.  Thanks.

**Comment by jimjamz** [comment]
@seal331 helpfully provided me links to the relevant documentation.
